libs/openFrameworks/gl/ofLight.cpp

```diff
@@ -224,7 +224,7 @@ float ofLight::getSpotlightCutOff() const{
 
 //----------------------------------------
 void ofLight::setSpotConcentration( float exponent ) {
-    data->exponent = CLAMP(exponent, 0, 128);
+    data->exponent = ofClamp(exponent, 0, 128);
```

Comment:
how about std::clamp? it is smarter with types. actually perhaps ofClamp should be deprecated?

Reply:
ah! yes it's c++17, but I thought it was allowed for 0.12?